### PR TITLE
README corrected

### PR DESCRIPTION
**Pull request by innabelaya**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,20 +1,22 @@
-# Statistics counters BEM-library
+# Statistics counters BEM library
 
-The library provides site statistics counters to use in BEM-based project.
+A library for web analytics that provides site statistics counters to all BEM-based project.
```
Comment (remnev) [inline README.md]: Are you sure about `a` article? AFAIK in phrases like `The smth that does smth` uses `the` article.

File: README.md [hunk 2]
```diff
@@ -1,20 +1,22 @@
...
 
 ## Blocks inside
-  - Yandex-metrica
-  - ~~Google-analytics~~ (in progress)
+  - [yandex-metrica](#link_to_the_block_in_your_repo) — uses [Yandex.Metrica](https://metrica.yandex.com) service.
+  - ~~[google-analytics](#link_to_the_block_in_your_repo)~~ (in progress) — uses [Google Analytics](http://www.google.com/intl/en/analytics/?utm_expid=71218119-7.lBgmrTO8R3uEDwsxNxa_Nw.0) service.
```
Comment (remnev) [inline README.md]: I propose to replace link with `http://www.google.com/analytics`

File: README.md (hunk 1 above)
Comment (innabelaya) [inline README.md]: You talk about the library for the very first time. You don't mention the definite BEM project or the names of the analytics tools. So, it's better to choose the indefinite article in this case.

File: README.md (hunk 2 above)
Comment (innabelaya) [inline README.md]: Yep, agree

File: README.md (hunk 1 above)
Comment (remnev) [inline README.md]: You convinced me :)

**Comment by remnev** [comment]
:+1:

File: README.md (hunk 1 above)
Comment (zxqfox) [inline README.md]: all → any?

File: README.md [hunk 3]
```diff
@@ -28,12 +30,12 @@ function getLevels(config) {
...
 ```
 
-bem-tools example
+*bem-tools example*
 ```js
 // TODO
 ```
 
-3. Declare any library block in your bemjson
+Declare the library block in your project BEMJSON:
```
Comment (zxqfox) [inline README.md]: Any of library blocks? `The library block` sounds like a steel.

File: README.md (hunk 1 above)
Comment (remnev) [inline README.md]: @innabelaya can you fix all to any?
>A library for web analytics that provides site statistics counters to any BEM-based project.

Comment (innabelaya) [inline README.md]: yes, I can :)

File: README.md (hunk 3 above)
Comment (remnev) [inline README.md]: @innabelaya can you rephrase the sentence like "Declare a counter in your project BEMJSON:"

Comment (innabelaya) [inline README.md]: Yes, sounds better!

**Comment by innabelaya** [comment]
@remnev done!

**Comment by remnev** [comment]
:+1:

**Comment by remnev** [comment]
@innabelaya thank you very much
